homekit/StateManager.js

```diff
@@ -42,6 +42,66 @@ function sanitize(service, characteristic, value) {
 	return value
 }
 
+function updateClimateReact() {		
+	device.state.smartMode.enabled = device.state.active && device.state.mode !== 'AUTO'
+	device.state.smartMode.type = 'temperature'
+	device.state.smartMode.highTemperatureWebhook = null	
+	device.state.smartMode.lowTemperatureWebhook = null	
+
+	if (device.state.mode === 'COOL') {
+		device.state.smartMode.highTemperatureThreshold = device.state.targetTemperature
+		device.state.smartMode.highTemperatureState = { 
+			on: true,
+			targetTemperature: device.state.targetTemperature,
+			temperatureUnit: device.temperatureUnit,
+			mode: device.state.mode,
+			fanSpeed: device.state.fanSpeed,
+			swing: device.state.swing,
```

Comment:
I moved to using verticalSwing
```suggestion
			swing: device.state.verticalSwing,
```